01/app.js

```diff
@@ -0,0 +1,3 @@
+const btn = document
+  .querySelector("button")

```

Comment:
Warto sprawdzić czy `button` został wyszukany - jeśli nie, to będzie błąd.
Oczywiście teraz wiemy, że on jest. Jednak w realnym projekcie może zostać usunięty i taki błędny kod spowoduje, że cały JS przestanie działać na stronie.